### ISSUE TITLE
No more calendar notifications

### ISSUE THREAD

**Issue by schiessle** [Freeyourgadget/Gadgetbridge#1525]
#### Before reporting a bug, please confirm the following:
- [x] I have read the [wiki](https://github.com/Freeyourgadget/Gadgetbridge/wiki), and I didn't find a solution to my problem / an answer to my question.
- [x] I have searched the [issues](https://github.com/Freeyourgadget/Gadgetbridge/issues), and I didn't find a solution to my problem / an answer to my question.
- [x] If you upload an image or other content, please make sure you have read and understood the [github policies and terms of services](https://help.github.com/articles/github-terms-of-service/#1-responsibility-for-user-generated-content)

#### Your issue is:

I don't know when it exactly broke, but it must be one of the last versions. Currently I'm using version 0.33.1 from f-droid together with the Amazfit Bip, firmware 1.1.5.16. The Android phone runs LineageOS 15 with microG.

I get all notifications but no calendar notifications. I already tried multiple calendar apps, but all with the same result. No notification if a calendar event happens. I'm not aware that I changed any settings nor could I found any settings which might be the reason that calendar notifications are blocked.

Any idea what could be the reason?

#### Your wearable device is:

*Please specify model and firmware version if possible*

#### Your Gadgetbridge version is:

*New issues about already solved/documented topics could be closed without further comments. Same for too generic or incomplete reports.*

**Comment by vanous** [comment]
Hello, is this still happening? Perhaps same issue as here? https://github.com/Freeyourgadget/Gadgetbridge/issues/1444

If already solved, please consider closing this issue, thank you

**Comment by schiessle** [comment]
I still don't get any calender notifications, meanwhile LineageOS 16 with microG and Gadgetbridge 0.63

**Comment by vanous** [comment]
I have tested calendar notifications (stock Samsung Google calendar app) and have received both:
- an event notifications (pre event,  on event) 
- a reminder (special type of event)

**Comment by schiessle** [comment]
What's also interesting in my case. I receive calendar notifications for tasks but not for "normal" calendar entries.

**Comment by vanous** [comment]
Does the calendar generate notification on the phone? If not, GB cannot forward it to the device...

**Comment by schiessle** [comment]
yes, the notification appears on the phone but GB doesn't forward it to the device.
